packages/contentful/migrations/gp2/projects/20240109100241-add-opportunities-related-fields.js

```diff
@@ -0,0 +1,65 @@
+module.exports.description =
+  'Updates projects with additional opportunities related fields';
+
+module.exports.up = (migration) => {
+  const projects = migration.editContentType('projects');
+
+  projects
+    .createField('opportunitiesAvailable')
+    .name('Opportunities Available')
+    .type('Boolean')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .defaultValue({
+      'en-US': false,
+    })
+    .disabled(false)
+    .omitted(false);
+
+  projects
+    .createField('opportunitiesShortText')
+    .name('Opportunities Short Text')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  projects
+    .createField('opportunitiesLinkName')
+    .name('Opportunities Link Name')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  projects.editField('pmEmail').required(true);
```

Comment:
I wonder if we should make this required because in prod there's a lot of projects without project PM email filled
![Screenshot 2024-01-15 at 11 22 56](https://github.com/yldio/asap-hub/assets/16595804/51283e0c-9470-4ddc-a88a-cdad9d4fcce2)